product-details.html

```diff
@@ -0,0 +1,459 @@
+            </ul>
+          </div>
+        </div>
+        <div class="details-right">
+          <div class="product-details">
+            <h2 class="product-details-heading">Product Details</h2>
+            <ul class="details-list">
+              <li class="details-item">
+                <div class="details-content">
+                  <div class="details-content-left">
+                    <img
+                      src="#"
+                      alt="DetailsIcon"
+                      class="details-content-img"
+                    />
+                    <p class="details-content-label">Created</p>
+                  </div>
+                  <div class="details-content-right">
+                    <p class="details-content-text">September 1, 2024</p>
+                  </div>
+                </div>
+              </li>
+              <li class="details-item">
+                <div class="details-content">
+                  <div class="details-content-left">
+                    <img
+                      src="#"
+                      alt="DetailsIcon"
+                      class="details-content-img"
+                    />
+                    <p class="details-content-label">File Size</p>
+                  </div>
+                  <div class="details-content-right">
+                    <p class="details-content-text">120 MB</p>
+                  </div>
+                </div>
+              </li>
+              <li class="details-item">
+                <div class="details-content">
+                  <div class="details-content-left">
+                    <img
+                      src="#"
+                      alt="DetailsIcon"
+                      class="details-content-img"
+                    />
+                    <p class="details-content-label">File Dimensions</p>
+                  </div>
+                  <div class="details-content-right">
+                    <p class="details-content-text">1920x1080 px</p>
+                  </div>
+                </div>
+              </li>
+              <li class="details-item">
+                <div class="details-content">
+                  <div class="details-content-left">
+                    <img
+                      src="#"
+                      alt="DetailsIcon"
+                      class="details-content-img"
+                    />
+                    <p class="details-content-label">Compatible App</p>
+                  </div>
+                  <div class="details-content-right">
+                    <p class="details-content-text">Figma, Sketch, Adobe XD</p>
+                  </div>
+                </div>
+              </li>
+            </ul>
+          </div>
+          <div class="details-product-tags">
+            <h2 class="product-tags">Product Tags</h2>
+            <ul class="product-tags-list">
+              <li class="product-tag-item">
+                <p class="product-tag">Real Estate</p>
```

Comment:
Usually tags it's a links. 
https://www.uinkits.com/components/tags-ui-element#:~:text=What%20are%20Tags%20in%20UI,have%20linked%20the%20tag%20with. 

> Action tags
> Action tags have a similar usage as buttons, as they can trigger certain actions by being pressed, sending the user to another page, or opening up a widget. However, as opposed to buttons, action tags are more compact, and that gives you the ability to be placed in a row.